### ISSUE TITLE
Strange extra line when creating universal box with 2 eye hinges

### ISSUE THREAD

**Issue by mdiabolo** [florianfesti/boxes#372]
Hi,

I did try to generate a basic box with 2 eyes hinges (mostly with default parameters)

[https://www.festi.info/boxes.py/UniversalBox?FingerJoint_angle=90.0&FingerJoint_style=rectangular&FingerJoint_surroundingspaces=2.0&FingerJoint_edge_width=1.0&FingerJoint_extra_length=0.0&FingerJoint_finger=2.0&FingerJoint_play=0.0&FingerJoint_space=2.0&FingerJoint_width=1.0&Stackable_angle=60&Stackable_height=2.0&Stackable_holedistance=1.0&Stackable_width=4.0&Hinge_grip_percentage=0&Hinge_outset=0&Hinge_outset=1&Hinge_pinwidth=0.5&Hinge_style=outset&Hinge_axle=2&Hinge_grip_length=0&Hinge_hingestrength=1&CabinetHinge_bore=3.2&CabinetHinge_eyes_per_hinge=10&CabinetHinge_hinges=2&CabinetHinge_style=outside&CabinetHinge_eye=1.5&CabinetHinge_play=0.05&CabinetHinge_spacing=2.0&Lid_angle=90.0&Lid_hole_width=0&Lid_second_pin=0&Lid_second_pin=1&Lid_spring=both&Lid_style=rectangular&Lid_surroundingspaces=2.0&Lid_edge_width=1.0&Lid_extra_length=0.0&Lid_finger=3.0&Lid_play=0.05&Lid_space=2.0&Lid_width=1.0&Click_angle=5.0&Click_bottom_radius=0.1&Click_depth=3.0&RoundedTriangleEdge_height=150.0&RoundedTriangleEdge_r_hole=2.0&RoundedTriangleEdge_radius=30.0&RoundedTriangleEdge_outset=1&Flex_stretch=1.05&Flex_connection=1.0&Flex_distance=0.5&Flex_width=5.0&top_edge=k&bottom_edge=h&x=100.0&y=50&h=50&outside=0&outside=1&vertical_edges=finger+joints&lid=default+%28none%29&thickness=3.0&format=svg&tabs=0.0&debug=0&labels=0&labels=1&reference=100&burn=0.1&render=0](url)

But I got strange extra lines:
![image](https://user-images.githubusercontent.com/48786714/159162506-dc711447-2368-49e2-bc23-7e31809ebbbb.png)

In Inkscape this looks strange too:
![image](https://user-images.githubusercontent.com/48786714/159162541-ad8dff8e-9f13-42f4-b443-e69e85d0138c.png)

**Comment by florianfesti** [comment]
Yeah, this is a bit of a hack. This hinge is part of the top edge that only starts after the corner that moves invards by the width of the edges of the sides. The hinge protrudes back out to the left and right that is beyond the with of the top edge.

Nowadays there is a solution for this problem but is requires special handling. I need to check whether I want to add that to the UniversalBox or move this out into its own generator.
